llvm/test/DebugInfo/X86/convert-loclist.ll

```diff
@@ -1,8 +1,15 @@
-; RUN: llc -mtriple=x86_64 -filetype=obj < %s \
+; RUN: llc -mtriple=x86_64-linux -filetype=obj < %s \
```

Comment:
Don't add `-linux`. `x86_64` is the generic ELF triple. It's preferred (short) and makes it clear that is a property shared among all ELF OSes.